pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -544,16 +526,6 @@ func handleCreate(ctx *zedmanagerContext, key string,
 	}
 	publishAppInstanceStatus(ctx, &status)
```

Comment:
Old code checked for status,Error != "' here. Why do we no longer need that check?